### DISCUSSION TITLE
I think I'm doing something wrong sveltekit clear my cookies

### DISCUSSION THREAD

**Discussion by Kokoserver** [sveltejs/kit#7336] [Q&A]
Sveltekit clear my cookies when I redirect to external URL or from an external URL to sveltekit. I'm using have tried window.open and window.location.href still the same thing.

**Comment by Hejtmus** [comment]
> Sveltekit clear my cookies when I redirect to external URL or from an external URL to sveltekit. I'm using have tried window.open and window.location.href still the same thing.

Hello,
this can be caused by number of reasons, but I don't think it is caused by SvelteKit. Can you please send snippet of code, how you create that cookie? Also it would be useful to know where do you host your SvelteKit app, I had similar problem, where Firebase hosting was clearing all cookies.

**Reply by Kokoserver** [reply]
I was able to solved it. This happened anytime I set my cookies to samesite:strict. By removing it everything works

**Comment by uhillbom** [comment]
Experience same thing that the cookie is cleared out after some time. 30 - 60 seconds. I need to login again.
I use samesite: strict at the moment for the cookie set but this is something that occurs recently and worked for a long time before.
I use netlify as host.
Could this be a bug?

**Reply by Kokoserver** [reply]
So the problem is if the {samesite:strict}, is quite self-explanatory immediately you redirect to the site from another site or to another site the same site is no longer through because you have moved to another site. @Rich-Harris is this true
